firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -354,6 +354,10 @@ class BoolSetting: Setting, FeatureFlaggable {
         return defaultValue
     }
 
+    func getPrefs() -> Prefs? {
+        return prefs
+    }
```

Comment:
I've seen this pattern in a few other places and it doesn't seem like idiomatic Swift to me; kind of curious what the rationale behind this is or what benefit it provides. (No objections to us doing this btw, just unclear why this is better than exposing the getter for `prefs` directly)

Reply:
I kept consistent with what the code is, but agreed in aligns more on what I'm used to with Java instead of Swift. maybe this can be a discussion in terms of styling with the team. 

I removed the `private` since a `private(set)` is only if we're setting the value. Let me know if you were thinking about exposing a different way.